capable/Cargo.toml

```
@@ -1,15 +1,23 @@
 [package]
 name = "mc-sgx-capable"
+# When updating version, update the deps.rs link in README.md
 version = "0.1.0"
 edition = "2021"
 authors = ["MobileCoin"]
 rust-version = "1.62.1"
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/mobilecoinfoundation/sgx"
-description = '''
-This crate exposes an idiomatic rust interface to the SGX SDK's `libsgx_capable` libraries.
-'''
+description = "Rust wrappers for the `sgx_capable` library."
+
+categories = ["external-ffi-bindings", "hardware-support"]
```

:upside_down_face: Looking at https://crates.io/categories it seems that `external-ffi-bindings` are the ffi crates, while `api-bindings` are the idiomatic crates.
```suggestion
categories = ["api-bindings", "hardware-support"]
```